doc/src/manual/arrays.md

```diff
@@ -528,10 +528,8 @@ applied to arrays (and combinations of arrays and scalars) as a
 operations like `<`, *only* the `.<` version is applicable to arrays.)
 
 1. Unary arithmetic -- `-`, `+`, `!`
-2. Binary arithmetic -- `+`, `-`, `*`, `/`, `\`, `^`, `.^`, `div`, `mod`
+2. Binary arithmetic -- `+`, `-`, `*`, `/`, `\`, `^`, `.^`
 3. Comparison -- `==`, `!=`, `≈` ([`isapprox`](@ref)), `≉`
-4. Unary Boolean or bitwise -- `~`
-5. Binary Boolean or bitwise -- `&`, `|`, `⊻` ([`xor`](@ref))
 
 Some operators without dots operate elementwise anyway when one argument is a scalar:
 `*`, `+`, `-`, and the bitwise operators. The operators `/` and `\` operate elementwise when
```

Comment:
maybe list the bitwise operators here now that the bullets for them are being deleted